packages/validation/src/user.ts

```diff
@@ -0,0 +1,3 @@
+export const USER_SOCIAL_WEBSITE = /^http(s?):\/\/.+/i;
+export const USER_SOCIAL_RESEARCHER_ID = /^[a-z]{1,3}-\d{4}-20[0-2]\d$/i;
+export const USER_SOCIAL_NOT_URL = /^(?!\s*http(s?):\/\/).+/i;
```

Comment:
`USER_SOCIAL_NOT_WEBSITE` would be consistent with the other thing, and more correct since other URLs are matched.
It's good that they are prefixed with USER_SOCIAL I think, the regexes as per requirements are hardly well-thought out so this method of 'website checking' should stay local to user socials :D